lib/playground.dart

```diff
@@ -357,7 +357,7 @@ class Playground {
   }
 
   void _handleHelp() {
-    if (context.focusedEditor == 'dart' && _isDocPanelOpen) {
+    if (context.focusedEditor == 'dart' && _isDocPanelOpen && editor.document.getSelection.length == 0) {
```

Comment:
Instead of `getSelection.length == 0`, you can use `getSelection.isEmpty`, which is slightly more semantic.